StreamVideoSwiftUITests/Utils/ReusePool_Tests.swift

```diff
@@ -0,0 +1,111 @@
+//
+//  ReusePool_Tests.swift
```

Comment:
Stream header

Reply:
Oh that's interesting. I'm wondering with swift-format didn't catch it